bin/start-worker

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-celery -A posthog worker
+# start celery worker with heartbeat (-B)
+celery -A posthog worker -B
```

Comment:
Does this run redbeat too?

Reply:
Nope, this is without. I thought about adding it here, but for local development with one worker there's no need for it. Then again, it's probably best to have as much parity with production as possible... so, [added](https://github.com/PostHog/posthog/pull/494/commits/5c507ee33757ccc9d9fd198ccdf517c3ab4b2088).